### ISSUE TITLE
Cannot install 0.3.0 or 0.3.1

### ISSUE THREAD

**Issue by LuciaTajuelo** [iwpnd/flashgeotext#40]
Hi:

I try to install flashgeotext 0.3.1 via pip or pipenv but i get the following error:

COMAND:
pipenv iinstall flashgeotext~=0.3.0

ERROR:
ERROR: Could not find a version that satisfies the requirement flashgeotext~=0.3.0
ERROR: No matching distribution found for flashgeotext~=0.3.0

How can i solve it?

Thanks in advance.
Best.

**Comment by iwpnd** [comment]
Hi @LuciaTajuelo, tested both `poetry` and `pip`

```bash
pip install flashgeotext==0.3.1
```

```bash
poetry install flashgeotext
# or
poetry add flashgeotext@latest
```

all of them work.
I'm not aware of the version syntax you're using `~=0.3.0`. I don't have much to investigate otherwise. 

Regards

**Comment by LuciaTajuelo** [comment]
Hi @iwpnd , thanks for your quick replay!

I've tried to upgrade flashgeotext using pip, please find attached a screenshot of the error.


![error_pip](https://user-images.githubusercontent.com/45468250/114549510-624e6e00-9c61-11eb-9f50-a7c79b73787c.png)

**Comment by iwpnd** [comment]
Any chance you can test it for `0.2.0` so I can make sure that I didn't screw anything up publishing with poetry for the first time?

**Comment by iwpnd** [comment]
Ah, found the problem.

flashgeotext >0.2.0 requires you to use python 3.8+, something I did when I moved to `poetry` for package management. I'll fix this in version 0.3.2, as it works for python 3.6+ also

**Comment by iwpnd** [comment]
`pip install flashgeotext==0.3.2` should work for you now. Closing this. Thanks for pointing it out. 🙇‍♂️ 🙏
